app/javascript/components/physical-storage-form/physical-storage-form.schema.js

```diff
@@ -0,0 +1,75 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+function createSchema(data) {
+  return {
+    fields: [
+      {
+        component: componentTypes.SELECT,
+        name: 'ems_id',
+        id: 'ems_id',
+        label: __('Provider:'),
+        options: data.provider.options,
```

Comment:
DDF supports the loading of options dynamically by passing a `loadOptions` function that returns a promise that resolves to an array of `{label, value}` entities.